aspnetcore/release-notes/aspnetcore-5.0.md

```diff
@@ -0,0 +1,476 @@
+
+![swagger/index.html view](~/release-notes/static/swagger-ui-page-rc1.png)
+
+## Blazor
+
+### `JSObjectReference` replacement
+
+`JSObjectReference` has been made internal, and `IJSObjectReference` takes its place in the public API. For more information, see the [official announcement (aspnet/Announcements #435)](https://github.com/aspnet/Announcements/issues/435) and <xref:blazor/call-javascript-from-dotnet?view=aspnetcore-5.0#blazor-javascript-isolation-and-object-references>.
+
+### CSS isolation
+
+Blazor now supports defining CSS styles that are scoped to a given component. Component specific CSS styles make it easier to reason about the styles in an app and to avoid unintentional side effects from global styles. <!-- UNCOMMENT CROSS-LINK AFTER https://github.com/dotnet/AspNetCore.Docs/pull/19956 MERGES: For more information, see <xref:blazor/components/css-isolation>.-->
+
+### New `InputFile` component
+
+The `InputFile` component allows reading one or more files selected by a user to be upload. For more information, see <xref:blazor/file-uploads>.
+
+### New `InputRadio` and `InputRadioGroup` components
+
+Blazor has built-in `InputRadio` and `InputRadioGroup` components that simplify data binding to radio button groups with integrated validation. For more information, see <xref:blazor/forms-validation>.
+
+### Component virtualization
+
+Improve the perceived performance of component rendering using the Blazor framework's built-in virtualization support. For more information, see <xref:blazor/components/virtualization>.
+
+### `ontoggle` event support
+
+Blazor events now support the `ontoggle` DOM event. For more information, see <xref:blazor/components/event-handling#event-argument-types>.
+
+### Set UI focus in Blazor apps
+
+Use the `FocusAsync` convenience method on element references for setting the UI focus on that element.
+<!-- UNCOMMENT CROSS-LINK WHEN https://github.com/dotnet/AspNetCore.Docs/issues/20232 IS WORKED. For more information, see <xref:blazor/components/event-handling#{ADD BOOKMARK}>.-->
+
+### Custom validation class attributes
+
+Custom validation class names are useful when integrating with CSS frameworks, such as Bootstrap. For more information, see <xref:blazor/forms-validation#custom-validation-class-attributes>.
+
+### IAsyncDisposable support
+
+Blazor components now support the <xref:System.IAsyncDisposable> interface for the asynchronous release of allocated resources.
+
+### JavaScript isolation and object references
+
+Blazor enables JavaScript isolation in standard [JavaScript modules](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Modules). For more information, see <xref:blazor/call-javascript-from-dotnet#blazor-javascript-isolation-and-object-references>.
+
+### Influence the HTML head in Blazor apps
```

Comment:
These components didn't end up getting finalized for .NET 5, so this section should be removed.

Reply:
@danroth27 ... We placed 5.0 content into the topics, so I'll comment it out in-place on a new PR. It was reviewed and is good coverage, so we'll have it in-place for 5.x later.